<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeachingGraphDash</class>
 <widget class="QDialog" name="TeachingGraphDash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCustomPlot" name="bar_graph" native="true">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>3000</width>
       <height>3000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="searchIn">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Search Member Name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fromCB"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="toCB"/>
     </item>
     <item>
      <widget class="QComboBox" name="progLevel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Undergraduate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Postgraduate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Continuing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pieChart">
       <property name="text">
        <string>Pie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="graphBtn">
       <property name="text">
        <string>Graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="print_bar_Button">
       <property name="text">
        <string>Print Bar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="print_pie_Button">
       <property name="text">
        <string>Print Pie</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
